homeassistant/components/amcrest/binary_sensor.py

```diff
@@ -48,111 +47,93 @@ class AmcrestSensorEntityDescription(BinarySensorEntityDescription):
 SCAN_INTERVAL = timedelta(seconds=BINARY_SENSOR_SCAN_INTERVAL_SECS)
 _ONLINE_SCAN_INTERVAL = timedelta(seconds=60 - BINARY_SENSOR_SCAN_INTERVAL_SECS)
 
-_AUDIO_DETECTED_KEY = "audio_detected"
-_AUDIO_DETECTED_POLLED_KEY = "audio_detected_polled"
-_AUDIO_DETECTED_NAME = "Audio Detected"
-_AUDIO_DETECTED_EVENT_CODES = {"AudioMutation", "AudioIntensity"}
+AUDIO_DETECTED_KEY = "audio_detected"
+AUDIO_DETECTED_POLLED_KEY = "audio_detected_polled"
+AUDIO_DETECTED_NAME = "Audio Detected"
+AUDIO_DETECTED_EVENT_CODES = {"AudioMutation", "AudioIntensity"}
 
-_CROSSLINE_DETECTED_KEY = "crossline_detected"
-_CROSSLINE_DETECTED_POLLED_KEY = "crossline_detected_polled"
-_CROSSLINE_DETECTED_NAME = "CrossLine Detected"
-_CROSSLINE_DETECTED_EVENT_CODE = "CrossLineDetection"
+CROSSLINE_DETECTED_KEY = "crossline_detected"
+CROSSLINE_DETECTED_POLLED_KEY = "crossline_detected_polled"
+CROSSLINE_DETECTED_NAME = "CrossLine Detected"
+CROSSLINE_DETECTED_EVENT_CODE = "CrossLineDetection"
 
-_MOTION_DETECTED_KEY = "motion_detected"
-_MOTION_DETECTED_POLLED_KEY = "motion_detected_polled"
-_MOTION_DETECTED_NAME = "Motion Detected"
-_MOTION_DETECTED_EVENT_CODE = "VideoMotion"
+MOTION_DETECTED_KEY = "motion_detected"
+MOTION_DETECTED_POLLED_KEY = "motion_detected_polled"
+MOTION_DETECTED_NAME = "Motion Detected"
+MOTION_DETECTED_EVENT_CODE = "VideoMotion"
```

Comment:
Do all these need to be renamed? Can we leave the `_` to make this merge smaller?